torch/_inductor/pattern_matcher.py

```diff
@@ -1103,6 +1181,140 @@ def normalize_args(**kwargs):
         return pattern.pattern
 
 
+_serialized_patterns: Set[str] = set()
+
+
+def _serialize_pattern(
+    unique_name: str,
+    search_fn,
+    example_inputs: Iterable[Any],
+    trace_fn: Callable[[Callable[..., Any], Iterable[Any]], torch.fx.GraphModule],
+    scalar_workaround,
+):
+    def get_file_template() -> str:
+        auto_generated_msg = textwrap.dedent(
+            """\
+            # This is an auto-generated file. Please do not modify it by hand.
+            # To re-generate, run:
+            # cd ~/pytorch && python
+            # torchgen/fuse_attention_patterns/gen_attention_patterns.py
```

Comment:
The message is relevant - `gen_attention_patterns` will rebuild ALL the patterns (including the non-fuse ones).  If you want I can rename that script to something else but I worried that something/someone might be referencing that script so figured it would be easier to just have that script do all the work.